docker-app/qfieldcloud/subscription/models.py

```diff
@@ -839,6 +839,12 @@ def create_subscription(
             )
             # NOTE to get annotations, mostly `is_active`
             trial_subscription_obj = cls.objects.get(pk=trial_subscription.pk)
+
+            if created_by.is_person and created_by.remaining_trial_organizations > 0:
+                # TODO Fix: for unknown reason created_by is sometimes an Organization
```

Comment:
TODO (see comment)

Reply:
@suricactus do you know by channce, why created_by is an 'Organization' here? I am really a bit stuck here ...